dtbook-utils/src/main/resources/xml/dtbook-load.xpl

```diff
@@ -87,7 +88,34 @@
             </p:add-attribute>
         </p:viewport>
     </p:group>
-    <p:identity name="fileset"/>
+    <p:identity name="dtbook-resources-mathml"/>
     <p:sink/>
-
+    
+    <!-- add any linked CSS stylesheets -->
+    <p:for-each>
+        <p:iteration-source>
+            <p:pipe step="main" port="source"/>
+        </p:iteration-source>
+        <p:try>
+            <p:group>
+                <px:xml-to-css-fileset include-links="true"/>
+            </p:group>
+            <p:catch>
+                <px:message message="CSS stylesheet URI(s) are malformed." severity="WARNING"/>
```

Comment:
shouldn't this catch block be included in the `px:xml-to-css-fileset` step itself? it would make its use more streamlined.

Reply:
This code was taken from dtbook-to-daisy3: https://github.com/daisy/pipeline-scripts/pull/151/files#diff-ac59fab3e4d8ca5d9a283d765dad2644L117. Looking at the step now, I don't even know if such an error will ever be thrown from px:xml-to-css-fileset. But yes you are right that not throwing the error would make its use more streamlined. Although in general I would say it's better to throw an error and to catch it then to not throw an error at all. You can also have an option to choose whether errors are ignored or not but I find that less logical.

So maybe it's more an issue of the awkward XProc syntax?